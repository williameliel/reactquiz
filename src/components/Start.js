import React from 'react';

const Start = ({title, description, handleStart}) => {

  return (
    <div>
      <h2>{title}</h2>
      <p>{description}</p>
      <button 
        className="btn btn-default" 
        onClick={handleStart}>Start</button>
    </div>
  );
};

export default Start;
