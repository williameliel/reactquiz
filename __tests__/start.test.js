import React from 'react';
import Start from '../src/components/Start';
import renderer from 'react-test-renderer';

it('Start renders correctly', () => {
  const tree = renderer.create(
    <Start />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
