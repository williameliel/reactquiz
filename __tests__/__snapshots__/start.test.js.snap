// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Start renders correctly 1`] = `
<div>
  <h2 />
  <p />
  <button
    className="btn btn-default"
    onClick={undefined}
  >
    Start
  </button>
</div>
`;
